src/lib/staged_ledger/inputs.ml

```diff
@@ -127,6 +135,6 @@ module type S = sig
   end
 
   module Transaction_witness :
-    Coda_pow.Transaction_witness_intf
-    with type sparse_ledger := Sparse_ledger.t
+    Transaction_witness_intf
+    with type sparse_ledger := Sparse_ledger.Stable.V1.t
```

Comment:
I think this one can stay `Sparse_ledger.t`, because the type `sparse_ledger` doesn't have `bin_io`.